spec/features/log_in_spec.rb

```diff
@@ -0,0 +1,28 @@
+# As a Maker
+# So that only I can post messages on Chitter as me
+# I want to log in to Chitter
+
+feature 'log in' do
+  scenario 'a user can log in' do
+    User.create(email: "test@example.com", password: "password123", username: "test_user")
+
+    visit('/sessions/new')
```

Comment:
You could use a helper for the log in process to avoid repeating yourself in these tests